crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -2290,14 +2250,14 @@ impl AuthorityPerEpochStore {
         certificate: &VerifiedExecutableTransaction,
```

Comment:
same question, can this function be deleted? (and the interface of the one in `SharedObjVerManager` updated to return what's needed in the one place from which it's used?)

an additional irony here is that the above TODO is already done in the inner function that this function wraps